bin/runtime/src/lib.rs

```diff
@@ -236,7 +238,7 @@ parameter_types! {
     pub const MaxLocks: u32 = 50;
     // We have only 2 reasons for holds - CodeUploadDeposit and StorageDeposit.
     pub const MaxHolds: u32 = 2;
-    pub const MaxFreezes: u32 = 0;
+    pub const MaxFreezes: u32 = 1;
```

Comment:
Maybe we want here 50 too, I think there is no downside in setting it high. Do you have a reference point maybe on how Polkadot runtime has it, or some parachain? I think we will set `MaxHolds` to 50 in one of hotfixes too.

Reply:
Here - https://github.com/polkadot-fellows/runtimes/blob/main/relay/polkadot/src/lib.rs#L323
They use value 8 here :)

Reply:
Set to 50.